crates/accelerate/src/sabre_swap/sabre_dag.rs

```diff
@@ -13,6 +13,7 @@
 use hashbrown::HashSet;
 use pyo3::prelude::*;
 use rustworkx_core::petgraph::prelude::*;
+use std::collections::HashMap;
```

Comment:
Should we use hashbrown here too just for the faster hashing?

Reply:
Good catch. Fixed in 5d9f647.